plugins/maven/src/main/java/org/wildfly/swarm/plugin/maven/StartMojo.java

```diff
@@ -328,7 +329,7 @@ protected SwarmExecutor executor(final Path appPath, final String name,
                         Files.createTempFile(Paths.get(this.projectBuildDir), "swarm-", "-cp.txt").toFile() :
                         Files.createTempFile("swarm-", "-cp.txt").toFile();
 
-                tmp.deleteOnExit();
+                getPluginContext().put("swarm-cp-file", tmp);
```

Comment:
Awesome @Ladicek. I didn't take that into account. Fixed and (again) updated pull.